src/Sylius/Bundle/ApiBundle/Doctrine/QueryItemExtension/OrderShopUserItemExtension.php

```diff
@@ -25,8 +25,10 @@
 /** @experimental */
 final class OrderShopUserItemExtension implements QueryItemExtensionInterface
 {
-    public function __construct(private UserContextInterface $userContext)
-    {
+    public function __construct(
+        private UserContextInterface $userContext,
+        private array $nonFilteredCartAllowedOperations = [],
```

Comment:
```suggestion
        private array $allowedNonGetOperations = [],
```
or just
```suggestion
        private array $allowedOperations = [],
```
?
the current name is, well, weird 🤷

Reply:
I would be for adjusting this name, especially since that parameter seems to have a more meaningful name

Reply:
I know, but it's hard to pick a meaningful name. `allowedOperations` might suggest the operations to which the extension should be applicated. However, it affects only the "filter cart" part :/.